### PR TITLE
Adds arccos, arcsin, arctan aliases 

### PR DESCRIPTION
**Pull request by mruberry**
These aliases are consistent with NumPy (see, for example, https://numpy.org/doc/stable/reference/generated/numpy.arccos.html?highlight=acos). 

Note that PyTorch's existing names are consistent with Python (see https://docs.python.org/3.10/library/math.html?highlight=acos#math.acos) and C++ (see, for example, https://en.cppreference.com/w/cpp/numeric/math/acos).

### CODE REVIEW

File: torch/csrc/jit/passes/normalize_ops.cpp [hunk 1]
```diff
@@ -15,7 +15,13 @@ static const std::unordered_map<Symbol, Symbol> alias_map = {
     {aten::linalg_det, aten::det},
     {aten::outer, aten::ger},
     {aten::arccosh, aten::acosh},
-    {aten::arccosh_, aten::acosh_}};
+    {aten::arccosh_, aten::acosh_},
+    {aten::arccos, aten::acos_},
```
Comment (gchanan) [inline torch/csrc/jit/passes/normalize_ops.cpp:19]: shouldn't this be the non-inplace variant?

**Review by gchanan — APPROVED** [review]
looks good once JIT issue is addressed.

File: torch/csrc/jit/passes/normalize_ops.cpp (hunk 1 above)
Comment (mruberry) [reply torch/csrc/jit/passes/normalize_ops.cpp:19]: Yep. Good catch.

Comment (mruberry) [reply torch/csrc/jit/passes/normalize_ops.cpp:19]: Also: let's elaborate the test to catch errors like this just in case they occur again.

**Comment by vfdev-5** [comment]
@mruberry I wonder why `arc*` alias methods are defined here as 
```c++
// arcsin, alias of asin
Tensor& arcsin_out(Tensor& result, const Tensor& self) { return unary_op_impl_out(result, self, asin_stub); }
Tensor arcsin(const Tensor& self) { return unary_op_impl(self, at::asin_out); }
Tensor& arcsin_(Tensor& self) { return unary_op_impl_(self, at::asin_out); }
```
using `unary_op_impl_` vs, for example, `clip`
```c++
// Implements the "clip" alias for clamp
Tensor& clip_out(Tensor& result, const Tensor& self, optional<Scalar> min, optional<Scalar> max) {
  return at::clamp_out(result, self, min, max);
}
Tensor clip(const Tensor& self, optional<Scalar> min, optional<Scalar> max) {
  return at::clamp(self, min, max);
}
Tensor& clip_(Tensor& self, optional<Scalar> min, optional<Scalar> max) {
  return at::clamp_(self, min, max);
}
```
thanks !

**Comment by mruberry** [comment]
> @mruberry I wonder why `arc*` alias methods are defined here as
> 
> ```c++
> // arcsin, alias of asin
> Tensor& arcsin_out(Tensor& result, const Tensor& self) { return unary_op_impl_out(result, self, asin_stub); }
> Tensor arcsin(const Tensor& self) { return unary_op_impl(self, at::asin_out); }
> Tensor& arcsin_(Tensor& self) { return unary_op_impl_(self, at::asin_out); }
> ```
> 
> using `unary_op_impl` vs, for example, `clip`
> 
> ```c++
> // Implements the "clip" alias for clamp
> Tensor& clip_out(Tensor& result, const Tensor& self, optional<Scalar> min, optional<Scalar> max) {
>   return at::clamp_out(result, self, min, max);
> }
> Tensor clip(const Tensor& self, optional<Scalar> min, optional<Scalar> max) {
>   return at::clamp(self, min, max);
> }
> Tensor& clip_(Tensor& self, optional<Scalar> min, optional<Scalar> max) {
>   return at::clamp_(self, min, max);
> }
> ```
> 
> thanks !
> 
> Context:
> I have an issue with dispatching on `arcsin.out` which does not dispatch as `asin.out`...

Hey @vfdev-5, sorry to hear you're hitting an issue. I agree having them just immediately call the asin functions would be simpler. I'll put up a PR for that now.

**Comment by vfdev-5** [comment]
@mruberry thanks ! Otherwise I can also send a PR with the fix if you want...

**Comment by mruberry** [comment]
> @mruberry thanks ! Otherwise I can also send a PR with the fix if you want...

Thank you but I rolled the update into a PR adding a couple more aliases. See https://github.com/pytorch/pytorch/pull/43762. It should land soon.

**Comment by mruberry** [comment]
Should be OK now, @vfdev-5 -- let me know.

**Comment by vfdev-5** [comment]
@mruberry thanks for the PR, it fixes my issue !

**Comment by mruberry** [comment]
> @mruberry thanks for the PR, it fixes my issue !

Awesome!

**Pull request closed without merging**